slash/react/src/Form/Checkbox/CheckboxInput.tsx

```diff
@@ -25,30 +29,34 @@ const CheckboxInput = forwardRef<HTMLInputElement, Props>(
       className,
       forceDisplayMessage,
       required,
+      disabled = false,
       ...checkboxProps
     },
     inputRef,
   ) => {
-    let rowModifier = classModifier;
-    if (mode === CheckboxModes.classic) {
-      rowModifier += " label-top";
-    }
-    if (required) {
-      rowModifier += " required";
-    }
+    const rowModifier = `${classModifier ?? ""}${
```

Comment:
Pas besoin vu qu'il a désormais une valeur par défaut avec un empty string
```suggestion
    const rowModifier = `${classModifier}${
```

Reply:
C'est modifié, et sur le textboxinput aussi où il y avait un fallback équivalent.